packages/e2e/src/page-objects/contacts.page.ts

```diff
@@ -7,122 +7,144 @@ import { ChainablePromiseElement } from "webdriverio"
 import Page from "./page"
 
 class ContactsPage extends Page {
+  /**[Selector] New contact button on Contacts screen */
```

Comment:
Those comments are unnecessary, class name and method names are enough in my op

Reply:
I was thinking about keeping these as a additional help so it is easier to get back to E2E development after doing some other tasks..